spec/models/category_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.